frontend/src/metabase/css/admin.css

```diff
@@ -264,7 +264,6 @@
 }
 
 .UserActionsSelect {
-  padding-top: 1em;
```

Comment:
Remove `padding-top` as it creates a "strange" white space at the top of the popover menu